.github/workflows/ci_workflows.yml

```diff
@@ -74,7 +78,10 @@ jobs:
     - name: Install Python dependencies
       run: python -m pip install --upgrade tox codecov
     - name: Run tests
-      run: tox ${{ matrix.toxargs }} -e ${{ matrix.toxenv }} -- ${{ matrix.toxposargs }}
+      run: |
+        tox ${{ matrix.toxargs }} -e ${{ matrix.toxenv }} -- ${{ matrix.toxposargs }}
+        echo "ARCH "$ARCH_ON_CI
+        echo "IS_CRON "$IS_CRON
```

Comment:
I mainly added this to make sure they are set correctly. I can remove them if they look too ugly.